web-common/src/features/dashboards/leaderboard/LeaderboardMeasureSelector.svelte

```diff
@@ -72,9 +89,19 @@
       class="flex flex-row items-center ui-copy-muted"
       style:grid-column-gap=".4rem"
       in:send={{ key: "leaderboard-metric" }}
-      style:max-width="355px"
+      style:max-width="400px"
     >
-      <div class="whitespace-nowrap">Dimension Leaders by</div>
+      <SeachableFilterButton
+        selectableItems={availableDimensionLabels}
+        selectedItems={visibleDimensions}
+        on:itemClicked={toggleDimensionVisibility}
+        on:deselectAll={setAllDimensionsNotVisible}
+        on:selectAll={setAllDimensionsVisible}
+        label="Dimensions"
+        tooltipText="Choose dimensions to display"
+      />
+
+      <div class="whitespace-nowrap">sorted by</div>
```

Comment:
I found this text a bit confusing.

`All dimensions sorted by total sales` reads like I sort the leaderboard cards themselves, not the values within them. Not sure I have a solution for this.

Reply:
We could change this to "showing top values by [selection]"